docs: Add 143 troubleshooting section (#1425)
seqeralabs/docs: fusion_docs/troubleshooting/general.md

```groovy
process.containerOptions = '--ulimit nofile=1048576:1048576'
```

## Exit code `143` on Kubernetes

> Tasks fail with exit code `143` and the pipeline halts instead of retrying the affected task.

Author: ```suggestion
When a task fails with exit code `143`, pipelines halt instead of retrying the affected task.
```
